README.md

```diff
@@ -6,7 +6,12 @@
 [![Coverage Status](https://coveralls.io/repos/github/RETFU/RREST/badge.svg?branch=master)](https://coveralls.io/github/RETFU/RREST?branch=master)
 ![WIP](https://img.shields.io/badge/unstable-master-red.svg)
 ![PHP7 supported](http://php7ready.timesplinter.ch/RETFU/RREST/badge.svg)
-> Important: **RREST** is in active development. The API is not frozen and BC break can happen at any time.
+
+## Versions
+> Important: **RREST 2.0** is in active development. It doesn't cover RAML 1.0 completely.
```

Comment:
Add a link to the RAML parser that document what is supported would be cool

Reply:
It is us that might not support all RAML 1.0 at first.

Reply:
Before tagging 2.0 we will need to make clear of the limitations of the first scope.